Sources/ClientRuntime/Networking/Http/URLSession/FoundationStreamBridge.swift

```diff
@@ -272,6 +283,15 @@ class FoundationStreamBridge: NSObject, StreamDelegate {
                 if writeCount > 0 {
                     logger.info("FoundationStreamBridge: wrote \(writeCount) bytes to request body")
                     buffer.removeFirst(writeCount)
+                    // TICK - smithy.client.http.bytes_sent
+                    var attributes = Attributes()
+                    attributes.set(
+                        key: HttpMetricsAttributesKeys.serverAddress,
+                        value: serverAddress)
+                    telemetry.bytesSent.add(
+                        value: writeCount,
+                        attributes: attributes,
+                        context: telemetry.contextManager.current())
```

Comment:
`FoundationStreamBridge` is where bytes are written, so `bytes_sent` are measured here